docs/Concepts/Privacy/Privacy-Overview.md

```diff
@@ -3,33 +3,44 @@ description: Privacy
 
 # Privacy
 
-Privacy in Besu refers to the ability to keep transactions private between the involved participants. 
-Other participants cannot access the transaction content or list of participants.
+In Besu, privacy refers to the ability to keep transactions private between the involved
+participants. Other participants cannot access the transaction content or list of participants.
 
 !!! important
-    For production systems requiring private transactions: 
-        
-    * We recommend using a network with a consensus mechanism supporting transaction finality. For example, 
-    [IBFT 2.0](../../HowTo/Configure/Consensus-Protocols/IBFT.md).
-    * Orion must be [highly available and run in a separate instance to Besu](../../HowTo/Use-Privacy/Run-Orion-With-Besu.md).
-    
+
+    For production systems requiring private transactions:
+
+    * It is recommended you use a network with a consensus mechanism supporting transaction
+    finality, such as [IBFT 2.0](../../HowTo/Configure/Consensus-Protocols/IBFT.md).
+    * Orion must be [highly available and run in a separate instance to Besu].
+
 !!! important
+
     Using private transactions with [pruning](../Pruning.md) is not supported.
 
 ## Private Transaction Manager
 
-Besu uses a Private Transaction Manager to implement privacy. For example, [Orion](http://docs.orion.pegasys.tech).  
-Each Besu node that sends or receives private transactions requires an associated Orion node. 
+Besu uses a Private Transaction Manager, [Orion](http://docs.orion.pegasys.tech), to implement
+privacy. Each Besu node sending or receiving private transactions requires an associated Orion
+node.
 
 ![Orion Nodes](../../images/OrionNodes.png)
 
-Private transactions are passed from the Besu node to the associated Orion node. The Orion node
-encrypts and directly distributes (that is, point to point) the private transaction to Orion nodes 
-participating in the transaction. 
+Private transactions pass from the Besu node to the associated Orion node. The Orion node
+encrypts and directly distributes (that is, point-to-point) the private transaction to the Orion
+nodes participating in the transaction.
+
+By default, each participant in a privacy network uses its own Besu and Orion
+node. [Multi-tenancy](Multi-Tenancy.md) allows multiple participants to use the same Besu and Orion
+node.
 
 By default, each participant in a privacy network uses its own Besu and Orion
 node. [Multi-tenancy](Multi-Tenancy.md) allows multiple participants to use the same Besu and Orion
 node.
```

Comment:
Is this paragraph included twice now?  Could you check this by publishing locally?

Reply:
I had various merge conflicts occuring. I suspect this could have popped in while I was trying to fix it. Duplicate removed.

Reply:
Fixed.